src/darwin/Framework/CHIP/MTRAsyncWorkQueue.mm

```diff
@@ -160,6 +182,25 @@ - (void)markComplete
     _duplicateCheckHandler = nil;
 }
 
+- (NSString *)description
+{
+    NSString * state;
+    switch (_state) {
+    case MTRAsyncWorkItemMutable:
+        state = @"mutable";
+        break;
+    case MTRAsyncWorkItemComplete:
+        state = @"complete";
+        break;
+    case MTRAsyncWorkItemEnqueued:
+        state = @"enqueued";
+        break;
+    default:
+        return [NSString stringWithFormat:@"<%@ %tu running retry: %tu>", self.class, _uniqueID, self.retryCount];
+    }
+    return [NSString stringWithFormat:@"<%@ %tu %@>", self.class, _uniqueID, state];
```

Comment:
This is the example of uniqueID being logged but not in the same format as other places.

Reply:
The description is more for debugging than for logging, all the log statements explicitly format the work item id.